stable/grafana/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 name: grafana
-version: 4.1.3
-appVersion: 6.5.0
+version: 4.1.4
```

Comment:
This version probably would fail since I just approved / lgtm your another PR.

Reply:
done!